### ISSUE TITLE
focusHighlight addons page

### ISSUE THREAD

**Issue by brennanyoung** [nvaccess/nvda#13417]
Apologies if this is the wrong place to report this. No way to do so on the dedicated nvda addons repo.

I'd like to raise a small but important issue with the addons page for focusHighlight: https://addons.nvda-project.org/addons/focusHighlight.en.html

The problem is an error message which indicates a problem with the addon itself (see https://github.com/nvdajp/focusHighlight/issues/22), but it's really a minor documentation problem. 

The latest version mentioned on the addons page is 6.3, but the latest version (at time of writing) is 6.4, available at https://github.com/nvdajp/focusHighlight/releases/tag/6.4

If I understand correctly, NVDA now incorporates the focusHighlight functionality. 

If so, does this mean that the separate focusHighlight addon is now superfluous for newer versions of NVDA? 
Should the addon be categorized as "legacy". 

A few notes on the addon page dedicated to focusHighlight explaining that the functionality is now part of the default NVDA installation (if that is indeed the case) would be very helpful. Would also be good to mention which version of NVDA took over the feature, and to update the link to the latest version, because older installations might still need the addon. 

Thank you!

**Comment by CyrilleB79** [comment]
Hello

From NVDA side, this issue should be closed because that's out of the scope of NVDA field.
@seanbudd, @feerrenrut could you close it?

Any issues regarding an add-on should be reported in the GitHub repo of this add-on or directly to the author. This includes among others missing or incorrect information in the add-on's documentation. 

Any issue regarding the add-on community website should be reported on the nvda-addons mailing list, on the add-on's GitHub repo or directly to the add-on author. This includes among others:
* website documentation not up-to-date with respect to the add-on's documentation
* download link (version) not up-to-date
* wrong classification of the add-on, e.g. add-on should be in legacy section

Just to say that the Focus Highlight add-on still has features that do not belong to NVDA's Focus Highlight feature, e.g. change color or line style. Thus the inclusinon of Focus Highlighting capability in NVDA does not make this add-on a legacy one. however, I agree that an up-to-date download link should be available on the community website.

I have also commented in https://github.com/nvdajp/focusHighlight/issues/22.
